redash/handlers/query_results.py

```diff
@@ -64,7 +64,7 @@ def run_query_sync(data_source, parameter_values, query_text, max_age=0):
         return None
 
 
-def run_query(data_source, parameter_values, query_text, query_id, max_age=0):
+def run_query(data_source, parameter_values, query_text, query_id, max_age=0, parameter_schema={}):
```

Comment:
Mutable types such as dictionaries shouldn't be used for parameter defaults, since the dictionary is persisted between function calls, more [info here](https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html).

```suggestion
def run_query(data_source, parameter_values, query_text, query_id, max_age=0, parameter_schema=None):
    if parameter_schema is None:
        parameter_schema = {}
```